src/Tizen.NUI.Components/Attributes/ViewAttributes.cs

```diff
@@ -264,5 +280,89 @@ public override Attributes Clone()
             return new ViewAttributes(this);
         }
 
+        /// <summary>
+        /// Apply ViewAttributes to View.
+        /// </summary>
+        public virtual void ApplyToView(View view, ControlStates state = ControlStates.Normal)
+        {
+            if (view == null)
+            {
+                return;
+            }
+
+            if (Position != null)
+            {
+                view.Position = Position;
+            }
+            if (Size != null)
+            {
+                view.Size = Size;
+            }
+            if (MinimumSize != null)
+            {
+                view.MinimumSize = MinimumSize;
+            }
+            if (PositionUsesPivotPoint != null)
+            {
+                view.PositionUsesPivotPoint = PositionUsesPivotPoint.Value;
+            }
+            if (ParentOrigin != null)
+            {
+                view.ParentOrigin = ParentOrigin;
+            }
+            if (PivotPoint != null)
+            {
+                view.PivotPoint = PivotPoint;
+            }
+            if (WidthResizePolicy != null)
+            {
+                view.WidthResizePolicy = WidthResizePolicy.Value;
+            }
+            if (HeightResizePolicy != null)
+            {
+                view.HeightResizePolicy = HeightResizePolicy.Value;
+            }
+            if (SizeModeFactor != null)
+            {
+                view.SizeModeFactor = SizeModeFactor;
+            }
+
+            if (!appliedViews.ContainsKey(view))
+            {
+                appliedViews.Add(view, 0);
```

Comment:
Wonder if a better solution than appliedViews would be storing a flag with the View?  Then would not have to worry about managing the lifecyle of Views.